test/specs/collections/Message/Message-test.js

```diff
@@ -0,0 +1,61 @@
+import faker from 'faker';
+import React from 'react';
+const Simulate = React.addons.TestUtils.Simulate;
+import {Message} from 'stardust';
+
+describe('Message', () => {
```

Comment:
It is worth reading through these test cases and assertions to get an idea of what this component is doing and how it works.